Design_Database

yuki-noguchi opened the pull request:
# WHAT
データベースの設計

# WHY
必要なテーブル、カラム、アソシエーションを洗い出すため

yuki-noguchi committed:
create pull request

yuki-noguchi committed:
Edit READMR.md

yuki-noguchi committed:
Revert "Edit READMR.md"

This reverts commit 2b0e917a076c6ca9c208605aa2c75eda5b496ee2.

yuki-noguchi committed:
Edit README.md

yuki-noguchi committed:
Revert "Edit README.md"

This reverts commit f4703642d791afaa724f62931ed4b1299466aa17.

yuki-noguchi committed:
Edit README.md

yuki-noguchi committed:
Revert "Edit README.md"

This reverts commit 9178851d7972779b67343afd6474d4ec266b1d85.

yuki-noguchi committed:
Edit README.md

yuki-noguchi committed:
Revert "Edit README.md"

This reverts commit 59ee124703186218cfe0c0eab913a5cbd8c2d59b.

yuki-noguchi committed:
Edit README.md

motemotenten commented:
#what ↓　
# what
というようにマークダウンを効かせましょう

motemotenten commented on README.md:
```diff
@@ -1,24 +1,42 @@
+| Column     | Type     | Options                   |
+|:-----------|----------|---------------------------|
+| name       | string   | null: false, unique: true |
+| groupe_id  | integer  | foreign_key: true         |
```
user と group でそれぞれお互いのidを持たせる必要はありません。多対多のアソシエーションの理解が足りないようなので、現場のメンターに聞いてみましょう。

motemotenten commented on README.md:
```diff
@@ -1,24 +1,42 @@
+
+### Association
+* belongs_to :user
+* belomgs_to :groupe
```
index をどのカラムに貼るのかも記述しましょう

motemotenten commented:
rails updateなど、余計なブランチは削除しましょう

yuki-noguchi committed:
Delete unnecessary columns

Deleted groupe_id column from users table
Deleted user_id column from groupes table

yuki-noguchi committed:
Add Index

yuki-noguchi committed:
Fixed typos

motemotenten commented on README.md:
```diff
@@ -1,24 +1,43 @@
+* has_many :messages
+* has_many :groups, through: :group_users
+
+###Index
```
マークダウンこちら効いていません

motemotenten commented on README.md:
```diff
@@ -1,24 +1,43 @@
+* belongs_to :user
+* belongs_to :group
+
+### groups table
```
こちら＃＃の間違いではないでしょうか？

yuki-noguchi committed:
Fix typos

###Index → ### Index
## group_users table → ### group_users table
